src/coreclr/tests/src/JIT/SIMD/VectorUtil.cs

```diff
@@ -237,7 +239,7 @@ private static String GetLogFileName()
     public JitLog()
     {
         fileStream = null;
-        simdIntrinsicsSupported = Vector.IsHardwareAccelerated;
+        simdIntrinsicsSupported = Vector.IsHardwareAccelerated && (Sse2.IsSupported || AdvSimd.IsSupported);
```

Comment:
A potentially better fix than this is to also have `IsHardwareAccelerated` return `false` and for the remaining `GT_SIMD` nodes that haven't been ported to also return `nullptr` if `SSE2` or `AdvSimd` aren't supported.

Reply:
I think that would indeed be a better fix. Is there any reason that would be problematic?

Reply:
I can't think of any off the top of my head. It would also make the two paths more cohesive as things transition over.

Reply:
I've updated to check the "baseline" ISA (`SSE2` for x86/x64 and `AdvSimd` for ARM64) and to return `nullptr` if it was disabled.
The one exception is for `SIMDIntrinsicHWAccel` which will return a `false` node in that scenario to help with dead code elimination.